app-text/cherrytree/cherrytree-0.37.0.ebuild

```diff
@@ -1,55 +1,50 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
You should drop or split this commit into a separate one. EAPI-bump requires the ebuild to be revbumped in any way.

Rest looks good!